File: dmy_house_logs/fast/view_logs_tail.c
#include "tmpl_log_tail.h"

#define LOG_TAIL_LEN 1000
#define TMPL_NAME "view_logs_tail.html"

int
main (void)
{
  gen_tmpl_log_tail (LOG_TAIL_LEN, TMPL_NAME);
  return 0;
}

File: dmy_house_logs/perl/view_logs_tail.pl
#! /usr/bin/env perl

use strict;
use warnings;

use CGI::Fast;
use tmpl_log_tail;

my $LOG_TAIL_LEN = 1000;
my $TMPL_NAME = "view_logs_tail.html";

while (my $q = new CGI::Fast) {
    gen_tmpl_log_tail($LOG_TAIL_LEN, $TMPL_NAME, $q);
}
